src/dmd/dsymbolsem.d

```diff
@@ -1215,9 +1215,26 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
                 }
                 else
                 {
+                    import dmd.hdrgen : toCBuffer, HdrGenState;
+                    OutBuffer buff;
+                    HdrGenState hgs;
+                    toCBuffer(dsym, &buff, &hgs);
+                    if (buff.offset != 0)
+                    {
+                        version(Windows)
```

Comment:
I had to do this to get around the extra newline that appears on Windows when using Outbuffer.extractString

Reply:
If anybody has some knowledge on a better approach I will happily substitute this noobish approach